Meadowlark.net/Meadowlark.Net.Core/Backend/Postgresql/Db.cs

```diff
@@ -0,0 +1,101 @@
+using Meadowlark.Net.Core.Backend.Model;
+using Meadowlark.Net.Core.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Npgsql;
+using NpgsqlTypes;
+
+namespace Meadowlark.Net.Core.Backend.Postgresql;
+
+public static class Db
+{
+  // TODO: Get .env loading working
+  private static readonly string connectionString = "Server=localhost;Port=5432;User Id=postgres;Password=abcdefgh1!;Database=MeadowlarkNet;";
+
+  // static Db()
+  // {
+  // Load .env file with Postgresql username/password
+  // DotNetEnv.Env.TraversePath().Load();
+  // connectionString: $"Server=localhost;Port=5432;User Id={Environment.GetEnvironmentVariable("POSTGRES_USER")};Password={Environment.GetEnvironmentVariable("POSTGRES_PASSWORD")};Database=MeadowlarkNet;");
+  // }
+
+  private static async Task TryCreateTable(ResourceInfo resourceInfo)
+  {
+    // Note: ProjectName and ResourceName have been validated against the ApiSchema, so SQL injection is not a concern here
+    string schemaName = resourceInfo.ProjectName.Value.Replace("-", "");
+    string tableName = resourceInfo.ResourceName.Value.Replace("-", "");
```

Comment:
We'll need to have an in-depth conversation as a team and community about whether to build the tables in MetaEd, deploy at runtime this way, or have a C# app with this same type of code that can run at deploy time.